### PR TITLE
[HOTFIX] CI failed due to dns reverse lookup issue

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

This is a hotfix to fix the ci failure due to dns reverse lookup issue. Some of the code is copied from pulsar project.
https://github.com/lhotari/pulsar/commit/b4300db307f542d2ece7ad9f487f078d00210a27

### What type of PR is it?
[ Hot Fix ]

### Todos
* [ ] - Task

### What is the Jira issue?
* 

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? NO
* Is there breaking changes for older versions? No
* Does this needs documentation? NO

### CODE REVIEW

File: spark/interpreter/src/test/java/org/apache/zeppelin/spark/SparkShinyInterpreterTest.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void testSparkShinyApp() throws IOException, InterpreterException, Interr
     Thread.sleep(5 * 1000);
     // extract shiny url
     List<InterpreterResultMessage> resultMessages = context2.out.toInterpreterResultMessage();
-    assertEquals(1, resultMessages.size());
+    assertEquals(resultMessages + "", 1, resultMessages.size());
```
Comment (cuspymd) [inline spark/interpreter/src/test/java/org/apache/zeppelin/spark/SparkShinyInterpreterTest.java:105]: The result seems be the same as `resultMessages.toString()`, but wouldn't it be better to use a consistent expression with other code?

Comment (zjffdu) [reply spark/interpreter/src/test/java/org/apache/zeppelin/spark/SparkShinyInterpreterTest.java:105]: This change is only for diagnose to display the content of resultMessages when this assertion fails.

Comment (zjffdu) [reply spark/interpreter/src/test/java/org/apache/zeppelin/spark/SparkShinyInterpreterTest.java:105]: > The result seems be the same as `resultMessages.toString()`, but wouldn't it be better to use a consistent expression with other code?

Right, let me update it

**Comment by zjffdu** [comment]
Will merge it soon so that it won't block other PRs

**Pull request closed without merging**